custom_components/amplipi/media_player.py

```diff
@@ -127,14 +130,32 @@ def __init__(self, namespace: str, source: Source, streams: List[Stream], vendor
         self._unique_id = f"{namespace}_source_{source.id}"
         self._last_update_successful = False
         self._attr_device_class = MediaPlayerDeviceClass.SPEAKER
+        self._is_off = False
 
+    async def async_toggle(self):
+        if self._is_off:
+            await self.async_turn_on()
+        else:
+            await self.async_turn_off()
+
+    async def async_turn_on(self):
+        self._is_off = False
```

Comment:
We may need a comment here to explain why we just set _is_off to False. I think it has something to do with how the media players work right?